run-ilab.json

```diff
@@ -0,0 +1,65 @@
+{
+  "tags": {
+    "topology": "none"
+  },
+  "endpoints": [
+    {
+      "type": "remotehosts",
+      "remotes": [
+        {
+          "engines": [ { "role": "client", "ids": "1" } ],
+          "config": {
+            "host": "localhost",
+            "settings": {
+              "controller-ip-address": "10.26.8.21",
```

Comment:
I think that we would generally prefer not to "leak" addresses like this.

Reply:
I figure since this was in the 10.x space it really did not matter.  But I can delete